pkg/pb/inspector.proto

```diff
@@ -47,6 +47,33 @@ service PieceStoreInspector {
   rpc Dashboard(DashboardRequest) returns (DashboardResponse) {}
 }
 
+service IrreparableInspector {
+  // ListSegments returns damaged segments
+  rpc ListSegments(ListSegmentsRequest) returns (ListSegmentsResponse);
+}
+
+// ListSegments
+message ListSegmentsRequest {
+  int64 limit = 1;
+  int64 offset = 2;
+}
+
+message IrreparableSegment {
+  bytes encrypted_path = 1;
+  bytes segment_detail = 2;
+  int64 lost_pieces = 3;
+  int64 last_repair_attempt = 4;
+  int64 repair_attempt_count = 5;
+}
+
+message SegmentGroup {
+  repeated IrreparableSegment segments = 1;  
+}
+
+message ListSegmentsResponse {
+  bytes data = 1;
```

Comment:
Why is this bytes instead of repeated IrreparableSegment?

Reply:
I thought it might be good to marshal the segments before sending them back

Reply:
I thought it might be good to marshal the segments before sending them back

Reply:
unless they need signing or some special treatment, marshaling is what protocol buffers do :D